sdk/python/kfp/_client.py

```diff
@@ -141,12 +141,12 @@ def _load_config(self, host, client_id, namespace):
 
   def _is_iap_host(self, host, client_id):
     if host and client_id:
-      return re.match(r'\S+.endpoints.\S+.cloud.goog', host)
+      return re.match(r'\S+.endpoints.\S+.cloud.goog/pipeline', host)
```

Comment:
I think that even without `/pipeline` it's still an IAP host. We should show meaningful error if we are sure the user forgot to add `/pipeline` to the URI.